### DISCUSSION TITLE
How to connect to Hypixel

### DISCUSSION THREAD

**Discussion by devAshton** [PrismarineJS/mineflayer#2400] [Q&A]
I am attempting to allow my bot to login via Hypixel with a Microsoft account. I have tried various things and none of them have worked. I have put in the email to the account and the password like in the code below. Nothing seems to be working, is there anything specific I have to do other than what I have?

> 
> const bot = mineflayer.createBot({
>     username: 'MYEMAIL@gmail.com',
>     password: 'MYPASSWORD',
>     host: 'mc.hypixel.net',
>     auth: "microsoft",
> })
>

**Comment by amoraschi** [comment]
What do you mean by "not working", did you add an error and kicked listener?

**Reply by devAshton** [reply]
No, I do not know how to do that as I have never used Mineflayer. I ran the bot and tried adding it as a friend multiple times and it just wasnt in the server at all.

**Reply by amoraschi** [reply]
> No, I do not know how to do that as I have never used Mineflayer. I ran the bot and tried adding it as a friend multiple times and it just wasnt in the server at all.

Add
```js
bot.on('error', console.log)
bot.on('kicked', console.log)
```
After you create the bot

Btw, you can also join the Discord Server for faster replies
